frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -188,6 +195,34 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
             error: undefined,
             annotations,
           });
+
+          /**Presence/Absence mode */
```

Comment:
There are several problems with the following block (see related comments), but on a more general note: the block above this one is supposed to be moment when we "finally" set the state after retrieving task. Each time you call `setState`, React re-renders the page, so it's best not to do it several times in the same function where it is not useful.